resources/testnet/seeds.toml

```diff
@@ -0,0 +1,64 @@
+[[seed]]
```

Comment:
Didn't Sergey mention something about not having too many seeds?

Reply:
IIRC that suggestion was an experiment to solve slow testnet syncing.  (It didn't work.)  I see no problem with this many seeds.